gardening/demo/crm_lead.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="crm_lead_2" model="crm.lead">
+    <field name="name">Rake a garden</field>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="stage_id" ref="crm.stage_lead1"/>
+    <field name="expected_revenue">250.0</field>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="description"><![CDATA[<p>Hi!</p><p>Can you rake my garden this week end?</p><p>See you.<br></p>]]></field>
+    <field name="priority">2</field>
+  </record>
+  <record id="crm_lead_3" model="crm.lead">
+    <field name="name">Jane Graden's opportunity</field>
+    <field name="partner_id" ref="res_partner_8"/>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="email_from">janes.graden@example.com</field>
+    <field name="stage_id" ref="crm.stage_lead2"/>
+    <field name="phone">+32548979443</field>
+    <field name="expected_revenue">5000.0</field>
+    <field name="description"><![CDATA[<p>Hi,</p><p>Can you do my whole garden and plant new plants.</p><p><br></p><p>Thanks.</p>]]></field>
+    <field name="priority">3</field>
+  </record>
+  <record id="crm_lead_4" model="crm.lead">
+    <field name="name">Information</field>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="stage_id" ref="crm.stage_lead1"/>
+    <field name="won_status">pending</field>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="description"><![CDATA[<p>I want to know your rates for gardening.</p>]]></field>
+  </record>
+  <record id="crm_lead_5" model="crm.lead">
+    <field name="name">Jhon Gredan's opportunity</field>
+    <field name="partner_id" ref="res_partner_9"/>
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="email_from">jhon.gredan@example.com</field>
+    <field name="email_state">correct</field>
```

Comment:
As Dhrumil mentioned, try to avoid such field like email_state, probability, date_closed... not really useful for the demo